### ISSUE TITLE
6.5.z branch

### ISSUE THREAD

**Issue by mirzal** [SatelliteQE/nailgun#631]
Robottelo 6.5.z branch is created and Airgun got separate issue for appropriate branch, but I don't see anything like that for Nailgun.

So... we want to create 6.5.z branch here, right? When would be a good time to do that?
@san7ket @oshtaier @jyejare @sean797

**Comment by sean797** [comment]
I would say anytime :+1:

**Comment by ntkathole** [comment]
similar issue #580

**Comment by mirzal** [comment]
@ntkathole That's true. Github fooled me and shows only ticket creation time, not update time. #580 was created back in December, so I didn't think it might refer to 6.5.z branching.

Right now, we have three PRs open:

- #611 - tagged WIP, DO NOT MERGE, bug and 6.5. Seems to be stalled.
- #629 - depends on PRs that are work in progress right now, so I'd say it's safe to merge after branching
- #630 - seems safe, but might be disruptive - I vote for merge after branch

I asked about comments for 6.5 branch in all these PRs. In my opinion, only #611 might be required to be merged before branch - other two should be merged after branch.

If there is no activity in #611 in next day or two, I'll go ahead and create 6.5.z branch. It doesn't seem that we need to wait for anything in particular. Let me know if you disagree.

**Comment by mirzal** [comment]
Update after comments from specific issues:

- #611 - fixes bug that is here for very long time. Would be nice to have it merged before branch, or cherry-pick later on. Requires someone to pick it up. Remaining work is mostly about testing that this change doesn't break anything.
- #629 - should be merged before branch, or cherry-picked later.
- #630 - should be merged after branch

**Comment by mirzal** [comment]
There wasn't much activity in listed PRs in last week. We decided to branch today and will cherry-pick any changes later on.
